client/extensions/woocommerce/state/data-layer/request/actions.js

```diff
@@ -10,30 +10,73 @@ import {
 	WOOCOMMERCE_REQUEST,
 } from 'woocommerce/state/action-types';
 
-function _createRequestAction( method, siteId, path, body ) {
-	return {
+function _createRequestAction( method, siteId, path, body, onSuccessAction, onFailureAction ) {
```

Comment:
See https://github.com/Automattic/wp-calypso/pull/15094/files#r121912720 , it probably makes more sense to fix it here.